frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,14 @@
 import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
-
-class Comment extends React.Component {
-    //todo: add like on comments
-    render() {
-        return (
-            <div className="comment-div">
-                <br/>
-                <Link to={`/account/${this.props.comment.account}`}>{this.props.comment.account}</Link>: {this.props.comment.comment}
-                <div className="date-created">{new Date(this.props.comment.created).toUTCString()}</div>
-                <br/>
-            </div>
-        )
-    }
-}
-
-export default Comment;
-
-
+import { Link } from "react-router-dom"
+import { useEffect } from 'react'
+import './SCSS/Comment.scss'
+export const Comment = (props) => {
+    return (
+        <div className="comment-div">
+            <br />
+            <Link to={`/account/${props.comment.account}`}>{props.comment.account} </Link> {props.comment.comment}
+            <div className="date-created">{new Date(props.comment.created).toDateString()}</div>
```

Comment:
the precision now is day. an idea maybe would be to add a function to calculate how long it has been from today to the input date. for example:

`getTimeDifference(a date 10 seconds ago)` would return `10 seconds`
`getTimeDifference(a date 30 minutes)` would return `30 minutes` (dont care about seconds anymore)
`getTimeDifference(a date 1 hour ago )` would return `1 hour` (dont care about minutes anymore)
`getTimeDifference(a date 10 days ago )` would return `10 days` (dont care about hours anymore)
`getTimeDifference(a date 3 months ago)` would return `3 months` (dont care about days anymore)
`getTimeDifference(anything else)` would return `X years`

Reply:
Oh that's a good idea, I'll take a look at it.